### PR TITLE
Fix #5181 - add -Xss to java_options ignore

### PR DESCRIPTION
**Pull request by ericdotdata**
## before

```
> cat .jvmopts
cat .jvmopts
-Dfile.encoding=UTF8
-Xms1G
-Xmx5G
-Xss6M
-XX:ReservedCodeCacheSize=512m
-XX:+UseG1GC

❯ sbt -d -mem 5632 'testOnly foo'
[addSbt] arg = '-debug'
[addMemory] arg = '5632'
[addJava] arg = '-Xms5632m'
[addJava] arg = '-Xmx5632m'
[addJava] arg = '-Xss4M'
[addJava] arg = '-XX:ReservedCodeCacheSize=512m'
...
# Executing command line:
java
-Dfile.encoding=UTF8
-Xss6M
-XX:+UseG1GC
-Xms5632m
-Xmx5632m
-Xss4M
-XX:ReservedCodeCacheSize=512m
-jar
/Users/eric/.sdkman/candidates/sbt/1.4.6/bin/sbt-launch.jar
-debug
```

## after

```
❯ cat .jvmopts
-Dfile.encoding=UTF8
-Xms1G
-Xmx5G
-Xss6M
-XX:ReservedCodeCacheSize=512m
-XX:+UseG1GC
❯ sbt -d -mem 5632 'testOnly foo'
[addSbt] arg = '-debug'
[addMemory] arg = '5632'
[addJava] arg = '-Xms5632m'
[addJava] arg = '-Xmx5632m'
[addJava] arg = '-Xss4M'
[addJava] arg = '-XX:ReservedCodeCacheSize=512m'
...
# Executing command line:
java
-Dfile.encoding=UTF8
-XX:+UseG1GC
-Xms5632m
-Xmx5632m
-Xss4M
-XX:ReservedCodeCacheSize=512m
-jar
/Users/eric/.sdkman/candidates/sbt/1.4.6/bin/sbt-launch.jar
-debug
```

### CODE REVIEW

**Review by eed3si9n — APPROVED** [review]

**Comment by ericdotdata** [comment]
@eed3si9n just added the `sbt.bat` for windows changes

**Comment by ericdotdata** [comment]
@eed3si9n shit I missed something in the windows.bat converting to draft and I'll see about adding tests

File: integration-test/src/test/scala/ScriptTest.scala [hunk 1]
```diff
@@ -24,115 +36,106 @@ object SbtScriptTest extends SimpleTestSuite with PowerAssertions {
...
-  test("sbt --sbt-version") {
-    val out = sbtProcess("--sbt-version", "1.3.13", "compile", "-v").!!.linesIterator.toList
+  makeTest("sbt --sbt-version")("--sbt-version", "1.3.13", "compile", "-v") { out: List[String] =>
     assert(out.contains[String]("-Dsbt.version=1.3.13"))
-    ()
   }
 
-  test("sbt -mem 503") {
-    val out = sbtProcess("compile", "-mem", "503", "-v").!!.linesIterator.toList
+  makeTest("sbt -mem 503", "compile")("-mem", "503", "-v") { out: List[String] =>
```
Comment (er1c) [inline integration-test/src/test/scala/ScriptTest.scala:76]: typo

**Comment by ericdotdata** [comment]
@eed3si9n back to active PR & CI passing, LMKWYT about the test refactoring

**Comment by eed3si9n** [comment]
@ericdotdata I'm 👍 on test refactoring.

**Review by eed3si9n — APPROVED** [review]

**Pull request merged**